spec/features/authorization_editor_spec.rb

```diff
@@ -2,7 +2,9 @@
 
 RSpec.describe "a user" do
   before(:each) do
-    @person = create(:person)
+    @department = create(:department, name: "Police")
+    @person = create(:person, department_id: @department.id)
```

Comment:
Wherever a person's department is not necessary for a test, then I'd prefer that we not define it.